Makefile

```diff
@@ -34,7 +34,13 @@ makefmt:
 
 .PHONY: test
 test:
+ifeq ($(ARCH), ppc64le) 
```

Comment:
This is not really required, but its fine please add the `ARCH` also as a variable similar to file plugin